transforms/code/header_cleanser/python/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM docker.io/python:3.11.9-slim-bullseye
+
+RUN pip install --upgrade pip 
+
+# Install system dependencies, including libgomp1
+RUN apt-get update && apt-get install -y \
```

Comment:
if you move this after pytest install (to match other Dockerfiles) you will get better docker image caching and faster ci/cd